.github/workflows/maven-goal/action.yml

```
@@ -25,5 +25,7 @@ runs:
         java-version-file: .java-version
         distribution: ${{ inputs.distribution }}
         cache: 'maven'
-    - run: ${{ inputs.command }}
+    - run: "$COMMAND"
```

```suggestion
    - run: "${COMMAND}"
```

Minor: adding curly braces to slightly improve clarity, and do the same for other variables.